open_spiel/games/universal_poker_test.cc

```diff
@@ -550,6 +551,98 @@ void TestFCHPA() {
   }
 }
 
+void TestHoleIndexCalculation() {
+  auto check_index = [](std::string card_a, std::string card_b,
+                        int expected_index) {
+    int a = logic::CardSet(card_a).ToCardArray()[0];
+    int b = logic::CardSet(card_b).ToCardArray()[0];
+    int actual_index = GetHoleCardsReachIndex(a, b,
+        /*num_suits=*/4, /*num_ranks=*/13);
+    SPIEL_CHECK_EQ(actual_index, expected_index);
+  };
+
+  // Suit order is "shdc"
+  check_index("2s", "2h", 0);
+  check_index("2s", "2d", 1);
+  check_index("2s", "2c", 2);
+  check_index("2s", "3s", 3);
+  check_index("2s", "3h", 4);
+  // ...
+  check_index("2s", "Ac", 50);
+  check_index("2h", "2d", 51);
+  check_index("2h", "2c", 52);
+  // ...
+  check_index("Ad", "Ac", 1325);
+}
+
+std::string ReadSubgameReachProbs(const std::string& file_name) {
+  std::string dir = __FILE__;
+  dir.resize(dir.rfind("/"));
+  return file::ReadContentsFromFile(
+      absl::StrCat(dir, "/universal_poker/endgames/", file_name, ".txt"), "r");
+}
```

Comment:
Suggestion: maybe it's more readable to define the value of reach probabilities for each subgame instead of reading it from a file!

Reply:
I didn't want to do that because it's an array of 2652 values, a bit too much for a source file.